hokusai/cli/review_app.py

```diff
@@ -51,11 +57,10 @@ def delete(app_name, verbose):
   """Deletes the Kubernetes based resources defined in ./hokusai/{APP_NAME}.yml"""
   set_verbosity(verbose)
   command(
-    hokusai.k8s_delete,
+    hokusai.delete_review_app,
```

Comment:
deserves its own review app command, since now it does namespace deletion outside of k8s_delete.